meeteval/wer/__main__.py

```diff
@@ -312,20 +345,25 @@ def average(files, out):
     return _merge(files, out, average=True)
 
 
-def cli():
-    # Define argument parser and commands
-    parser = argparse.ArgumentParser()
-    parser.add_argument('--version', action='store_true', help='Show version')
-
-    # Logging and verbosity
-    logging.addLevelName(100, 'SILENT')     # Add a level that creates no output
-    parser.add_argument(
-        '--log-level', help='Log level', default='WARNING',
-        choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL', 'SILENT']
-    )
+class CLI:
```

Comment:
Does it make sense to move this class somewhere else and import it in all `__main__.py` files? It seems weird that the DER CLI script imports the WER CLI script. `meeteval.cli`?

Same applies to the other functions that are imported in the DER CLI.

Reply:
While the CLI class can be used for WER and DER, it is mainly designed for WER.
Since we don't know, how future interfaces look like, I thought, we use here the concept that is used in kaldi:
Define the function where it is used and import it, where it also works.

But I could also move the code to cli.py